charts/galoy/templates/jwt-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: jwt-secret
+  labels:
+    app.kubernetes.io/managed-by: Helm
```

Comment:
is there some label or annotation to add so that this secret is not deleted even if the chart is uninstall?

Reply:
Yes. I have added it.